android/src/main/java/com/thebluealliance/androidclient/helpers/MatchHelper.java

```diff
@@ -95,7 +96,7 @@ public static TYPE fromKey(String key) {
         SHORT_TYPES.put(MatchHelper.TYPE.SEMI, "sf");
         SHORT_TYPES.put(MatchHelper.TYPE.FINAL, "f");
 
-        LONG_TYPES = new HashMap<>();
+        LONG_TYPES = new HashMap<>(); // TODO: I18N
```

Comment:
"TODO" marks a searchable to-do comment.

IntelliJ's TODO tool window makes it easy to go through them all.

The Google style is "TODO(jerry):", where the name says who to ask for more information, not necessarily who'll fix it or who wrote the comment. In this case, the task is simply to internationalize the strings by putting them into resources.